### DISCUSSION TITLE
Can I assign one AGW to several PLMNs

### DISCUSSION THREAD

**Discussion by aseaudi** [magma/magma#15188] [Q&A] [answered]
Hello
I am exploring the possibility of assigning one AGW to several PLMNs, to have a sort of a neutral host.
Is this possible with Magma ?

**Comment by javieraubert** [comment, marked as answer]
Hi, I don't think it is possible, because you have one PLMN per Network and you have assign the AGW to the Network, but just one Network at the time.
